actions/run/dist/index.js

```diff
@@ -2901,26 +2902,40 @@ async function run() {
     if (flags) {
         args.push(...flags.split(' '));
     }
-    if (artifact) {
-        args.push('--artifact', `${earthfile}+${target}/`, `${artifactPath}`);
-    }
-    else {
-        args.push(`${earthfile}+${target}`);
-    }
     if (targetFlags) {
         args.push(...targetFlags.split(' '));
     }
-    core.info(`Running command: ${command} ${args.join(' ')}`);
-    const output = await spawnCommand(command, args);
-    const imageOutput = parseImage(output);
-    if (imageOutput) {
-        core.info(`Found image: ${imageOutput}`);
-        core.setOutput('image', imageOutput);
-    }
-    const artifactOutput = external_path_.join(earthfile, parseArtifact(output));
-    if (artifactOutput !== earthfile) {
-        core.info(`Found artifact: ${artifactOutput}`);
-        core.setOutput('artifact', artifactOutput);
+    core.info(`Filtered targets >> ${targets}`);
+    const targetsArr = targets.split(' ');
+    for (const tg of targetsArr) {
```

Comment:
This seems a bit verbose. Why not just split the targets and then map the result? We don't need to log anything here either.